config/flags/flags.go

```diff
@@ -330,6 +330,10 @@ var (
 		Name:  "exitwhensynced.epoch",
 		Usage: "Exits after synchronisation reaches the required epoch",
 	}
+	RPCEVMSimulationFlag = cli.BoolFlag{
+		Name:  "rpc.evmsimulation",
+		Usage: "EVM is used for RPC transaction simulation",
```

Comment:
We should be precise in the terminology here: an EVM is an abstract specification of Ethereum's Virtual Machine which is implemented by `geth` and Tosca through its `lfvm` implementation. 

The description of this flag should thus be something like "if enabled, the geth implementation of the EVM is used in RPC transaction simulation calls";